OpenSim/Common/Test/testSTOFileAdapter.cpp

```diff
@@ -318,27 +318,37 @@ int main() {
               << std::endl;
     testReadingWriting<SimTK::SpatialVec>();
 
-    std::cout << "Testing exception for reading an empty file"
-              << std::endl;
+    std::cout << "Testing exception for reading an empty file" << std::endl;
     std::string emptyFileName("testSTOFileAdapter_empty.sto");
     std::ofstream emptyFile(emptyFileName);
-    SimTK_TEST_MUST_THROW_EXC(FileAdapter::createAdapterFromExtension(emptyFileName)->read(emptyFileName), FileIsEmpty);
+    SimTK_TEST_MUST_THROW_EXC(
+            FileAdapter::createAdapterFromExtension(emptyFileName)
+                    ->read(emptyFileName),
+            FileIsEmpty);
     std::remove(emptyFileName.c_str());
+}
```

Comment:
Should there be a try/catch block here so the file gets deleted even if the `SimTK_TEST_MUST_THROW_EXC()` fails?

Reply:
I added a `FileRemover` class to handle this issue.

Reply:
Cool 😎